server/prometheus.go

```diff
@@ -14,6 +16,20 @@ import (
 	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
 )
 
+type ThresholdResponse struct {
```

Comment:
Add godoc in public symbols

Reply:
Done for all the the code I added or modified.